workbench-core/example/infrastructure/integration-tests/tests/datasets/delete.test.ts

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import ClientSession from '../../support/clientSession';
+import Dataset from '../../support/resources/datasets/dataset';
+import Setup from '../../support/setup';
+import HttpError from '../../support/utils/HttpError';
+
+describe('datasets delete integration test', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+
+  const fakeDataSetId: string = 'example-ds-badbadba-dbad-badb-adba-dbadbadbadba';
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('RemoveDataSets', () => {
```

Comment:
do we need a test for trying to delete a dataset with a datasetId that is not a UUID?

Reply:
because of the way the route authz is written, this is an authz test and not a datasets test, but can't hurt! done!